clients/python/llmengine/api_engine.py

```diff
@@ -12,12 +12,15 @@
 from llmengine.errors import parse_error
 
 SPELLBOOK_API_URL = "https://api.spellbook.scale.com/llm-engine/"
-LLM_ENGINE_BASE_PATH = os.getenv("LLM_ENGINE_BASE_PATH", SPELLBOOK_API_URL)
 DEFAULT_TIMEOUT: int = 10
 
 api_key = None
 
 
+def get_base_path() -> str:
+    return os.getenv("LLM_ENGINE_BASE_PATH", SPELLBOOK_API_URL)
```

Comment:
nit: since we're changing things with this var, should we rename SPELLBOOK as well?